src/Linux/Avalonia.LinuxFramebuffer/Input/LibInput/LibInputBackendOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Avalonia.LinuxFramebuffer.Input.LibInput;
+
+/// <summary>
+/// LibInputBackend Options.
+/// </summary>
+public sealed record class LibInputBackendOptions
+{
+    /// <summary>
+    /// Used internally to pass libinput context to <see cref="LibInputBackend.InputThread(object?)"/>.
+    /// </summary>
+    internal IntPtr LibInputContext { get; init; }
```

Comment:
What is the purpose of passing your own context? Won't our configuration code cause clashes anyway?

Reply:
Not having been able to find information on whether the libinput context could be created in a Thread other than the main one, I preferred to leave it as it was previously.

Reply:
What should I do?